pom.xml

```diff
@@ -204,7 +204,7 @@
     <plugin.scm.version>1.11.2</plugin.scm.version>
     <plugin.shade.version>3.2.2</plugin.shade.version>
     <plugin.source.version>3.2.1</plugin.source.version>
-    <plugin.surefire.version>2.22.2</plugin.surefire.version>
+    <plugin.surefire.version>3.0.0-M5</plugin.surefire.version>
```

Comment:
Is this upgrade required by junit 5?

Reply:
Actually 2.22.2 is ok for junit 5 but I found that 3.0.0-M3 fix some bug about junit 5. 
So I upgrade surefire to the latest version.

> [JUnit Runner that writes to System.out corrupts Surefire's STDOUT when using JUnit's Vintage Engine](https://issues.apache.org/jira/browse/SUREFIRE-1614)

Reply:
Thanks for clarification.